dask/dataframe/tests/test_io.py

```diff
@@ -401,6 +401,11 @@ def test_from_pandas_dataframe():
     assert len(ddf.divisions) == len(ddf.dask) + 1
     assert type(ddf.divisions[0]) == type(df.index[0])
     tm.assert_frame_equal(df, ddf.compute())
+    ddf = dd.from_pandas(df, chunksize=8)
+    assert len(ddf.dask) == 3
+    assert len(ddf.divisions) == len(ddf.dask) + 1
+    assert type(ddf.divisions[0]) == type(df.index[0])
+    tm.assert_frame_equal(df, ddf.compute())
 
```

Comment:
Ok. This is now a test. (For some reason, its not showing this diff is outdated... but it is... maybe I messed up something?)